### PR TITLE
fixes coverage calculations

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one E2E Test exists testing the new feature
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [x] The new version is deployed to the dev server "dev1" using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [ ] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh` and execute `npm run testpipeline -- --env EXECUTION_ENVIRONMENT=""` in dataland-frontend
- [x] After(!) the cypress tests have passed locally, execute the backend-e2e-tests `./gradlew dataland-e2etests:test`

### CODE REVIEW

**Comment by ppuetsch** [comment]
image

**Comment by ppuetsch** [comment]
https://sonarcloud.io/summary/new_code?id=d-fine_Dataland&branch=DALA-968_fix_sonar_coverage

File: build.gradle.kts [hunk 1]
```diff
@@ -107,7 +106,7 @@ tasks.jacocoTestReport {
         xml.required.set(true)
         csv.required.set(false)
     }
-    executionData.setFrom(fileTree(projectDir).include("*.exec"))
+    executionData.setFrom(fileTree(projectDir).include("**/*.exec"))
```
Comment (fsoffel) [inline build.gradle.kts:109]: Es entzieht sich meiner Kenntnis, wo sich die .exec Dateien befinden. Die CI versucht auf jeden Fall auch noch Dateien der Form ./*.exec zu kopieren. Die sollten da inbegriffen sein, oder?
Ansonsten würde **.exec die auch mit einschließen.

Comment (ppuetsch) [reply build.gradle.kts:109]: **/*.exec heißt: Jede .exec Datei die in einem beliebigen Subfolder liegt.
*.exec heißt: jede exec Datei die im Folder selbst liegt.

Einige Files liegen aber in Subfoldern.

**Pull request merged**